_episodes/05-access-data.md

```diff
@@ -442,3 +443,53 @@ b01.rio.to_raster("B01.tif")
 > ~~~
 > {: .language-python}
 {: .callout}
+
+> ## Exercise: Downloading data from a protected bucket
+>
+> Use `earthaccess` to download 10 files from the ASTGTM version 003 dataset.
+>
+> >## Solution
+> >
+> > ~~~
+> > import earthaccess
+> > auth = earthaccess.login()
+> > ~~~
+> > {: .language-python}
+> >
+> > ~~~
+> > Using .netrc file for EDL
+> > ~~~
+> > {: .output}
+> >
+> > ~~~
+> > results = earthaccess.search_data(
```

Comment:
Previously in the lesson we've been showing how to use pystac-client to search for STAC assets. I'm unsure if this exercise has enough introduction to the earthaccess library to be included as is, since learners won't have seen a prior example of earthaccess being used in the lesson.

Is it possible to change this exercise to use pystac-client instead @abarciauskas-bgse and only use earthaccess for login?